dockstore-webservice/pom.xml

```diff
@@ -988,6 +988,11 @@
             <groupId>com.fasterxml.jackson.datatype</groupId>
             <artifactId>jackson-datatype-jsr310</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.eclipse.jetty</groupId>
+            <artifactId>jetty-http</artifactId>
+            <version>9.4.41.v20210516</version>
```

Comment:
Why did you need to add this? It doesn't seem like you need it, but I could be reading it wrong.

Reply:
Ditto, seems unlikely to be needed. 
But if it is, then you should add it to the dependency management in bom-internal and tie it to the same version as this property https://github.com/dockstore/dockstore/blob/822cfa0f9355c387bff940345b8e21a26013ff61/bom-internal/pom.xml#L68

Reply:
There wasn't a 429 status code in the httpStatus class we normally use, so I used a different one. Then the build failed and said I needed to add that to the pom. I just made a new variable now though